src/openforms/submissions/templates/report/submission_report.html

```diff
@@ -6,7 +6,11 @@
 <html lang="{{ LANG }}" class="{{ theme.get_classname }}">
     <head>
         <meta charset="utf-8">
-        <title>{% block title %}{% trans "Summary PDF" %}{% endblock %}</title>
+        <title>{% block title %}
+            {% blocktrans with form_name=report.form.name trimmed %}
+                {{form_name}}: PDF report
```

Comment:
```suggestion
                {{ form_name }}: PDF report
```
it parses/compiles etc., but the style guide demands spaces between the curly braces and the variable :)

what's the reasoning behind the textual change of Summary PDF -> PDF report?